osbs/api.py

```diff
@@ -243,9 +243,10 @@ def _create_build_config_and_build(self, build_request, namespace):
                         raise OsbsException(
                             self._panic_msg_for_more_running_builds(build_config_name, builds))
                     else:
-                        build = builds[0].request
```

Comment:
When `BuildConfig` gets autoinstantiated, `.request` is `None` resulting in explicitly starting another build.